suitcase/hdf5.py

```diff
@@ -9,14 +9,23 @@
 
 __version__ = "0.2.2"
 
-def export(headers, filename):
+def export(headers, filename, stream_name=None, fields_unwanted=None, timestamps=True):
```

Comment:
Not a huge fan of making the time stamps optional, they just were super helpful in debugging something at CHX.